src/wallet/wallet.ts

```diff
@@ -1100,6 +1106,68 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     return parseInt(parts[5], 10);
   }
 
+  /**
+   * Helper method to get authority tx_outputs
+   * Uses the getTxOutputs API method to return one or many mint authorities
```

Comment:
one or many authorities* not only mint

Reply:
Yes, fixed in 5b788957fd1124ee7f4f2f51eda098737313bb7a